app/controllers/api/verify/complete_controller.rb

```diff
@@ -4,7 +4,7 @@ class CompleteController < Api::BaseController
       def create
         result, personal_key = Api::ProfileCreationForm.new(
           password: verify_params[:password],
-          jwt: verify_params[:details],
+          jwt: verify_params[:user_bundle_token],
```

Comment:
One advantage of a single-word property is that it avoids needing to convert between casing (snake_case to camelCase in the client, camelCase back to snake_case in API calls). Dunno that I'd want that influencing our implementation too much, but thought it worth highlighting. I wondered if "token" would suffice, though I do kinda like how we're qualifying _what sort of token_ it is.

Reply:
`usertoken`?